src/iconv.cr

```diff
@@ -8,7 +8,7 @@ struct Iconv
     original_from, original_to = from, to
 
     @skip_invalid = invalid == :skip
-    {% unless flag?(:freebsd) %}
+    {% unless flag?(:freebsd) || flag?(:musl) %}
```

Comment:
It's not out of scope, but iconv in musl-libc doesn't provide any solution for ignoring invalid bytes —FreeBSD provides `__iconv` to solve this.

I'm not sure what I can do. I could raise, but we couldn't compile anything then, because `ignore_invalid` is the default. Or maybe I should let it choke on invalid bytes, check Errno, and replace the invalid bytes manually? Something like to http://git.alpinelinux.org/cgit/aports/tree/main/musl/iconv.c?h=3.5-stable#n76 I suppose.